### PR TITLE
add me as a person

### PR DESCRIPTION
**Pull request by HamaBarhamou**
Hi! I have added myself to the member of those this project and I would like you to review it. Thank you!

### CODE REVIEW

File: _organizers/hama-barhamou.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+github: HamaBarhamou
+layout: base
+name: ISSAKA HAMA Barhamou
+photo_url: /static/img/organizers/barhamou.jpg
+role: member of the Website Committee
```
Comment (drewbrew) [inline _organizers/hama-barhamou.md:6]:
```suggestion
role: Website team
```

**Review by drewbrew — CHANGES_REQUESTED** [review]
It looks good. Would you mind cropping your profile picture to be square (same width as height)? Thanks!

image

File: _organizers/hama-barhamou.md (hunk 1 above)
Comment (HamaBarhamou) [reply _organizers/hama-barhamou.md:6]: ok well collected. I take note of the recommendations

**Comment by HamaBarhamou** [comment]
hi @drewbrew I have made the update

**Review by drewbrew — APPROVED** [review]

**Comment by drewbrew** [comment]
Merged. Thanks so much!

**Comment by HamaBarhamou** [comment]
ok thanks you

**Pull request merged**